IBAnimatable/AnimatableBarButtonItem.swift

```diff
@@ -41,6 +41,12 @@ open class AnimatableBarButtonItem: UIBarButtonItem, BarButtonItemDesignable, An
   @IBInspectable open var damping: CGFloat = CGFloat.nan
   @IBInspectable open var velocity: CGFloat = CGFloat.nan
   @IBInspectable open var force: CGFloat = CGFloat.nan
+  @IBInspectable open var _timingFunction: String = "" {
```

Comment:
This one should not be `open`. It only is used in Interface Builder. Please have a look at the comment around https://github.com/IBAnimatable/IBAnimatable/issues/218#issuecomment-230324948 . We don't want the `_` properties be used in programmatical APIs.